src/RootContainer.js

```diff
@@ -45,7 +113,13 @@ class RootContainer extends React.Component {
 				) : this.state.loading ? (
 					<Loading />
 				) : (
-					<ResultTable data={this.state.data} />
+					<ResultTable
+						handleClick={this.handleClick}
+						data={this.state.data}
+						sortedByOrganName={this.state.sortedbyOrganName}
+						sortedByCellAmount={this.state.sortedByCellAmount}
```

Comment:
This is kinda the extra mile from my side - Could you please update to use object destructuring for all state data members instead since the names are a bit long.